profiler/profiler.oql/src/org/netbeans/modules/profiler/oql/engine/api/impl/hat.js

```diff
@@ -27,6 +27,8 @@ var PrimitiveArrayInstance = Java.type("org.netbeans.lib.profiler.heap.Primitive
 var Field = Java.type("org.netbeans.lib.profiler.heap.Field");
 var FieldValue = Java.type("org.netbeans.lib.profiler.heap.FieldValue");
 var GCRoot = Java.type("org.netbeans.lib.profiler.heap.GCRoot");
+// Injected by environment (type lookup not possible in GraalJS
+// var ArrayDump = Java.type("org.netbeans.lib.profiler.heap.ArrayDump");
```

Comment:
commented out intended?

Reply:
Actually I replaced a reference to `org.netbeans.lib.profiler.heap.ArrayDump` in an `instanceof` check with a variable `ArrayDump`. This comment is a reminder, that there is an injected variable, that needs to be taken care of by the environment. The block where I added it would be the place where I would expect the import and so from there I can look in the environment.